Sources/PubNub/Errors/ErrorDescription.swift

```diff
@@ -29,44 +29,122 @@ import Foundation
 
 public struct ErrorDescription {
   public static let stringEncodingFailure: String = {
-    "`String(data:encoding:)` returned nil when converting JSON Data to a `String`"
+    return "`String(data:encoding:)` returned nil when converting JSON Data to a `String`"
   }()
 
   public static let defaultRecoverySuggestion: String = {
-    "No recover suggestion was provided."
+    return "No recover suggestion was provided."
   }()
 
   public static let missingCryptoKey: String = {
-    "Missing cipher key from `PubNubConfiguration`"
+    return "Missing cipher key from `PubNubConfiguration`"
   }()
 
   public static let rootLevelDecoding: String = {
-    "AnyJSON could not decode invalid root-level JSON object"
+    return "AnyJSON could not decode invalid root-level JSON object"
   }()
 
   public static let keyedContainerDecoding: String = {
-    "AnyJSON could not decode value inside `KeyedDecodingContainer`"
+    return "AnyJSON could not decode value inside `KeyedDecodingContainer`"
   }()
 
   public static let unkeyedContainerDecoding: String = {
-    "AnyJSON could not decode value inside `UnkeyedDecodingContainer`"
+    return "AnyJSON could not decode value inside `UnkeyedDecodingContainer`"
   }()
 
   public static let rootLevelEncoding: String = {
-    "AnyJSON could not encode invalid root-level JSON object"
+    return "AnyJSON could not encode invalid root-level JSON object"
   }()
 
   public static let pushNotEnabled: String = {
-    "Use of the mobile push notifications API requires Push Notifications which is not enabled for this subscribe key"
+    return "Use of the mobile push notifications API requires Push Notifications which is not enabled for this subscribe key"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 125 characters](https://app.codacy.com/app/PubNub/swift/pullRequest?prid=4513555)